examples/maven/maven.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -euo pipefail
+
+# Define variables.
+GIT_ROOT=$(git rev-parse --show-toplevel)
+DEFAULT_REPOSITORY=$(xxd -l 16 -c 16 -p < /dev/random)
+: "${REGISTRY:=ttl.sh}"
+: "${REPOSITORY:=$REGISTRY/$DEFAULT_REPOSITORY}"
+'\033[32m'
+'\033[0m'
+
+# Install shared tasks.
+kubectl apply -f "${GIT_ROOT}"/platform/vendor/tekton/catalog/main/task/git-clone/0.4/git-clone.yaml
+kubectl apply -f "${GIT_ROOT}"/platform/vendor/tekton/catalog/main/task/maven/0.2/maven.yaml
```

Comment:
This file is currently sitting in `platform/vendor/tekton/catalog/main/task/maven/maven.yaml`. That file probably needs to be moved into a subdirectory `0.2` to conform with the path given here and to be consistent with the other vendored task manifests.